tests/unit/test-class-wc-payments-features.php

```diff
@@ -227,6 +227,7 @@ public function test_is_woopay_express_checkout_enabled_returns_false_when_woopa
 	}
 
 	public function test_is_woopay_direct_checkout_enabled_returns_true() {
+		update_option( 'woocommerce_woocommerce_payments_settings', [ 'enabled' => 'yes' ] );
```

Comment:
It would be good to add a test case checking if DC is disabled if `['enabled' => 'false']`

Reply:
Good call out. Added in [a80bce2](https://github.com/Automattic/woocommerce-payments/pull/9203/commits/a80bce23ffaeeb67946813ddfb089bda5a256a00).